nbcollection/ci/merge_artifacts/template/notebook.html

```diff
@@ -9,6 +9,7 @@
 				<a target="_blank" href="https://github.com/{{ page.org_name }}/{{ page.repo_name }}">
                 	<img class="logo" src="../../images/Jspectra.svg" alt="JWST Data Analysis Notebooks"/>
                 </a>
+                <a href="../../index.html" ,="" class="button" style="color: white !important">Return to Main Page</a>
```

Comment:
What does `,=""` do?  It does look like valid HTML.

Reply:
Probably nothing - I copied it from an editor.  I'll remove it and test it out and see if there's a difference.